samples/graph-activity-feed/csharp/ActivityFeedNotification/Views/Shared/TeamNotification.cshtml

```diff
@@ -69,24 +69,27 @@
                 $(targetBox).show();
             });
         });
-        microsoftTeams.initialize();
+
+        microsoftTeams.app.initialize();
+
         var token = localStorage.getItem("accessToken");
         var username = "";
         var userId = "";
         var teamchannelID = "";
         var teamId = "";
-        microsoftTeams.getContext(function (context) {
-            teamchannelID = context.channelId;
-            username = context.upn;
-            userId = context.userObjectId;
-            teamId = context.groupId;
 
-            console.log(context);
+        microsoftTeams.getContext(function (context) {
```

Comment:
getContext is using the v1 method signature

Reply:
Removed the file as it was not used